src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -376,3 +379,26 @@ export function decodeScalar(
       : undefined;
   return isString(unwrapped) ? unwrapped : undefined;
 }
+
+export function downloadAsCsv(tableData, filename) {
+  const {meta, data} = tableData;
+  if (meta !== undefined) {
```

Comment:
You could early return instead.

```typescript
if (meta === undefined) {
  return;
}
```

Reply:
:+1: sure thing, but for my learnings what's the difference?

Reply:
I think we use `typeof(meta) === "undefined"` for checking undefined values

Reply:
It saves you a level of indentation.